pkg/rook/status.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2023 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package rook
+
+import (
+	"fmt"
+	"strings"
+
+	bash "github.com/rook/kubectl-rook-ceph/pkg/bash"
+	"github.com/rook/kubectl-rook-ceph/pkg/k8sutil"
+)
+
+var scriptPrintSpecificCRStatus = `
+kubectl -n %s get %s -ojson | jq --monochrome-output '.items[].status'
+`
+
+var getCrdList = `
+kubectl -n  %s get crd | awk '{print $1}' | sed '1d'
+`
+
+func PrintCustomResourceStatus(_ *k8sutil.Context, clusterNamespace string, arg []string) {
```

Comment:
Why isn't the context passed yet? The PR that defined the context was merged, right?
```suggestion
func PrintCustomResourceStatus(context *k8sutil.Context, clusterNamespace string, arg []string) {
```

Reply:
it is not used anywhere in the function so discarded it.

Reply:
removed